### PR TITLE
Santa Catarina & Londrina

### PR DESCRIPTION
**Pull request by Skippern**
WMS for Santa Catarina, and WMTS for Londrina-PR

### CODE REVIEW

**Comment by Skippern** [comment]
WMTS services not seem to be recognised, it is recognised in JOSM as `wmts:http://siglon.londrina.pr.gov.br/arcgis/rest/services/Imagens/Ortofotos_2011_Paranacidade/MapServer/WMTS/1.0.0/WMTSCapabilities.xml`

Support for WMTS services should be added.

File: sources/south-america/br/Rio_2013.geojson [hunk 1]
```diff
@@ -7,13 +7,611 @@
         "country_code": "BR",
         "name": "Rio Mosaic 2013",
         "available_projections": [
+<<<<<<< HEAD
```
Comment (tyrasd) [inline sources/south-america/br/Rio_2013.geojson]: what's up with those merge conflict markers?

**Comment by tyrasd** [comment]
WMTS sources have been added to the index as type *tms* layers by specifying the actual tile URLs instead of the WMTS GetCapabilities call ([example](https://github.com/osmlab/editor-layer-index/blob/gh-pages/sources/europe/it/SouthTyrolOrthofoto2014.geojson)).

File: sources/south-america/br/Rio_2013.geojson (hunk 1 above)
Comment (Skippern) [inline sources/south-america/br/Rio_2013.geojson]: I have no idea where that conflict comes from, the `Rio*` files was not edited, to what I know, only source of conflict I can think of is not syncing the repository before creating new files, which shouldn't be causing conflicts at all.

**Comment by Skippern** [comment]
@tyrasd so changing `type: wmts` to `type:tms` should solve the build problem?

**Comment by Skippern** [comment]
Been looking at your example, and the GetCapabilities call, and I have problems figuring out how the TMS call URL looks like

`<ResourceURL format="image/jpgpng" resourceType="tile" template="http://siglon.londrina.pr.gov.br/arcgis/rest/services/Imagens/Ortofotos_2011_Paranacidade/MapServer/WMTS/tile/1.0.0/Imagens_Ortofotos_2011_Paranacidade/{Style}/{TileMatrixSet}/{TileMatrix}/{TileRow}/{TileCol}"/>`

**Comment by tyrasd** [comment]
it's not that tricky:

* `{TileMatrix}/{TileRow}/{TileCol}` correspond to the x y and z indices: `{z}/{y}/{x}`
* for `{TileMatrixSet}` you have to look in the GetCapabilities call and see what they offer (look for the `<TileMatrixSet>` tags). To me, `GoogleMapsCompatible` sounds good.
* similarly, you can look up the available options for `{Style}`. Here, there only is a `default` style.

The final tms link then looks like: https://siglon.londrina.pr.gov.br/arcgis/rest/services/Imagens/Ortofotos_2011_Paranacidade/MapServer/WMTS/tile/1.0.0/Imagens_Ortofotos_2011_Paranacidade/default/GoogleMapsCompatible/{z}/{y}/{x} (to check you can paste this into the "custom layer" of the iD editor)

PS: since this server apparently supports https, it makes sense to use it.

File: sources/south-america/br/Rio_2013.geojson (hunk 1 above)
Comment (tyrasd) [inline sources/south-america/br/Rio_2013.geojson]: hmm, strange. Ok, I think I can merge the Londrina changes manually. Don't worry.

**Comment by Skippern** [comment]
Good, because I am heading out the door for a month business travel, and don't know how I will access internet for the next 31 days

**Comment by tyrasd** [comment]
I've cherry-picked, modified and merged this manually now. Please make sure to update/rebase your local repository before making additional changes.

**Comment by Skippern** [comment]
No problem, I Will not work with this repo for another month, at least, will Malé sure to rebase before doing anything more, I'll look into rebase as soon as possible

**Comment by Klumbumbus** [comment]
The attribution url for Santa Catarina is strange, it leads to an pfd file which contains only a heading.

**Comment by Skippern** [comment]
@Klumbumbus I have asked Cassio Eskelsen respond, as he supplied this link to me a few months back.

**Comment by xuiqzy** [comment]
The link still does not point to an attribution. Did you hear anything back @Skippern or is there a better link?

**Comment by Skippern** [comment]
Have had no more feedback on the issue. And at the moment my schedule is too packed to pursue it any further

**Pull request closed without merging**